public/javascripts/Admin/src/Admin.GSVCommentView.js

```diff
@@ -0,0 +1,68 @@
+function AdminGSVCommentView(admin) {
+    var self = {};
+    self.admin = admin;
+
+    var _init = function() {
+        self.panoProp = new PanoProperties();
+
+        _resetModal();
+    };
+
+    function _resetModal() {
+        var modalText =
+            '<div class="modal fade" id="labelModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">'+
```

Comment:
Can you actually also modify this HTML (and the same code in `GSVLabelView`) to user `this-case` instead of `camelCase` for element IDs like `labelModal` -> `label-modal`

Reply:
Have you heard that referred to as "kebab-case"? Fixed those instances

Reply:
I had not! Thank you :)